packages/commonwealth/server/passport/magic.ts

```diff
@@ -372,6 +421,16 @@ async function magicLoginRoute(
     }
     generatedAddresses.push({ address: cosmosAddress, chain: DEFAULT_COSMOS_CHAIN });
 
+    if (chainToJoin) {
```

Comment:
Remove this block.

Reply:
I think this is OK now because we've removed the old generatedAddresses.push() calls.

Reply:
Curious why this comment has not been resolved since Jun 1?